module/planning/PlanWalkPath/README.md

```diff
@@ -0,0 +1,28 @@
+# PlanWalkPath
+
+## Description
+
+Plans a walk path based on information given. Three types of walk path planning are in this module
+
+1. Walking towards a particular point. This uses a point in torso space and walks directly to it, ignoring obstacles.
+2. Turn on the spot. Without moving translationally, turn on the spot. Supports both directions.
```

Comment:
Isn't this more of a walk command than a plan?
I think maybe that it belongs in strategy

Reply:
they are all used by the WalkToBall strategy tho, which I don't know if I have a PR up for yet (should soon)